src/components/data_display/generateFilterDataFields/generateFilterDataFields.ts

```diff
@@ -87,12 +96,12 @@ export const generateFilterDataFields = async (table: string, fullUpdate: boolea
     }
 
     // Delete unused (no filter definitions) columns
-    for (const { name } of currentColumns) {
-      await db.dropColumn(tableNameFull, name)
-    }
+    // for (const { name } of currentColumns) {
+    //   await db.dropColumn(tableNameFull, name)
+    // }
```

Comment:
Not needed?

Reply:
Oops, yes that is needed. I just turned it off temporarily so it wouldn't keep deleting a column. I'll restore it.

Reply:
6a777798